app/main.py

```diff
@@ -1 +1,36 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(self, name: str, health: int = 100) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = False
+
+        Animal.alive.append(self)
+
+    def __repr__(self) -> str:
+        class_dict = {
+            "Name": self.name,
+            "Health": self.health,
+            "Hidden": self.hidden
+        }
+        rep = (f"{{"
+               f'Name: {class_dict["Name"]}, '
+               f'Health: {class_dict["Health"]}, '
+               f'Hidden: {class_dict["Hidden"]}'
+               f"}}")
+        return rep
```

Comment:
try to refactor this method in  a more more concise way. avoid creating additional dictionary and variable. simply return the required string